.github/workflows/build.yaml

```diff
@@ -238,4 +232,172 @@ jobs:
+
+      - name: Install cargo-deb
+        run: cargo install cargo-deb --no-default-features
+
+      - name: Get git tag version
+        run: |
+          git fetch --all --tags
+          echo "TAG=$(git tag | tail --lines=1)" >> $GITHUB_ENV
+
+      - name: Set release version
+        run: |
+          echo "RELEASE_VERSION=$(echo ${{ env.TAG }} | sed 's/v//')" >> $GITHUB_ENV
+
+      - name: Prebuild package Debian arm64
+        run: |
+          wget https://github.com/nanu-c/axolotl/archive/v${{ env.RELEASE_VERSION }}.tar.gz --output-document=$GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}.tar.gz
+          mkdir --parents $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/axolotl
+          cp --recursive $GITHUB_WORKSPACE/target/aarch64-unknown-linux-gnu/release/axolotl/* $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/axolotl/
+          cp $GITHUB_WORKSPACE/deb/LICENSE $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/LICENSE
+          cd $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }} && debmake --yes --email arno_nuehm@riseup.net --fullname "Arno Nuehm" --monoarch
+          sed -i '3d;4d' $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/debian/changelog
+          awk -i inplace 'NR == 3 {print "  * See upstream changelog below."} {print}' $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/debian/changelog
+          echo >> $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/debian/changelog
+          cat $GITHUB_WORKSPACE/docs/CHANGELOG.md >> $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/debian/changelog
+          sed -i 's/<preferred name and address to reach the upstream project>/Aaron <aaron@nanu-c.org>/' $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/debian/copyright
+          sed -i 's/<url:\/\/example.com>/https:\/\/github.com\/nanu-c\/axolotl/' $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/debian/copyright
+          cp $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/debian/changelog $GITHUB_WORKSPACE/deb/
+          cp $GITHUB_WORKSPACE/axolotl-${{ env.RELEASE_VERSION }}/debian/copyright $GITHUB_WORKSPACE/deb/
+
+      - name: Build package Debian arm64
+        run: cargo deb --target=aarch64-unknown-linux-gnu --no-build
+
+      - name: Upload package artifact Debian arm64
+        uses: actions/upload-artifact@v3
+        with:
+          name: Axolotl-Debian-arm64-package
+          path: target/aarch64-unknown-linux-gnu/debian/axolotl_*_arm64.deb
+          retention-days: 1
+
+  package-click:
+    strategy:
+      matrix:
+        arch: [armhf, arm64, amd64]
+    name: Package as click ${{ matrix.arch }}
+    runs-on: ubuntu-latest
+    needs:
+      - build-axolotl-web
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+
+      - name: Download axolotl-web build artifacts
+        uses: actions/download-artifact@v3
+        id: download-axolotl-web
+        with:
+          name: axolotl-web
```

Comment:
Is this enough or do we need additional artifacts?

Reply:
No, the axolotl-web is enough in this part